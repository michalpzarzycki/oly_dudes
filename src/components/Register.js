import { useState } from 'react';
import Avatar from '@material-ui/core/Avatar';
import Button from '@material-ui/core/Button';
import CssBaseline from '@material-ui/core/CssBaseline';
import TextField from '@material-ui/core/TextField';
import FormControlLabel from '@material-ui/core/FormControlLabel';
import Checkbox from '@material-ui/core/Checkbox';
import Link from '@material-ui/core/Link';
import Grid from '@material-ui/core/Grid';
import Box from '@material-ui/core/Box';
import LockOutlinedIcon from '@material-ui/icons/LockOutlined';
import Typography from '@material-ui/core/Typography';
import Container from '@material-ui/core/Container';
import styles from './Register.module.css'
import { signUpUserWithEmailAndPassword } from '../firebase/auth'
export default function Register() {
const [user, setUser] = useState({})
function handleChange(event) {
    setUser({...user, [event.target.name]: event.target.value})
}
async function handleSubmit(e) {
    e.preventDefault();
    const {email, password} = user;
    try{
        let user = await signUpUserWithEmailAndPassword(email, password);
        console.log("USER", user)
    }  catch(err) {
        console.log("ERROR", err)
    }
}
  return (
    <Container component="main" maxWidth="xs">
      <CssBaseline />
      <div className={styles.paper}>
        <Avatar className={styles.avatar} co>
          <LockOutlinedIcon />
        </Avatar>
        <Typography component="h1" variant="h5" className={styles.header}>
          Sign up
        </Typography>
        <form className={styles.form} noValidate onSubmit={handleSubmit}>
          <Grid container spacing={2}>
            <Grid item xs={12} sm={6}>
              <TextField
                autoComplete="fname"
                name="firstName"
                variant="outlined"
                required
                fullWidth
                id="firstName"
                label="First Name"
                autoFocus
                onChange={handleChange}
              />
            </Grid>
            <Grid item xs={12} sm={6}>
              <TextField
                variant="outlined"
                required
                fullWidth
                id="lastName"
                label="Last Name"
                name="lastName"
                autoComplete="lname"
                onChange={handleChange}
              />
            </Grid>
            <Grid item xs={12}>
              <TextField
                variant="outlined"
                required
                fullWidth
                id="email"
                label="Email Address"
                name="email"
                autoComplete="email"
                onChange={handleChange}
              />
            </Grid>
            <Grid item xs={12}>
              <TextField
                variant="outlined"
                required
                fullWidth
                name="password"
                label="Password"
                type="password"
                id="password"
                autoComplete="current-password"
                onChange={handleChange}
              />
            </Grid>
            <Grid item xs={12}>
              <FormControlLabel
                control={<Checkbox value="allowExtraEmails" color="primary" />}
                label="I want to receive inspiration, marketing promotions and updates via email."
              />
            </Grid>
          </Grid>
          <Button
            type="submit"
            fullWidth
            variant="contained"
            color="primary"
            className={styles.submit}
          >
            Sign Up
          </Button>
          <Grid container justify="flex-end">
            <Grid item>
              <Link href="#" variant="body2">
                Already have an account? Sign in
              </Link>
            </Grid>
          </Grid>
        </form>
      </div>
      <Box mt={5}>
      <Typography variant="body2" color="textSecondary" align="center">
      {'Copyright Â© '}
      <Link color="inherit" href="https://material-ui.com/">
        Your Website
      </Link>{' '}
      {new Date().getFullYear()}
      {'.'}
    </Typography>
      </Box>
    </Container>
  );
}