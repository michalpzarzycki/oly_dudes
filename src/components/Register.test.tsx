import {Register} from './Register';
import renderer from 'react-test-renderer';

it('render correctly Register component', () => {  
    const RegisterComponent = renderer.create(<Register />).toJSON();
    expect(RegisterComponent).toMatchSnapshot();
});