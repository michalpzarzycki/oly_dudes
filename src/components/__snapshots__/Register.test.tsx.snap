// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly Register component 1`] = `
<main
  className="MuiContainer-root MuiContainer-maxWidthXs"
>
  <div
    className="paper"
  >
    <div
      className="MuiAvatar-root MuiAvatar-circle avatar MuiAvatar-colorDefault"
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M18 8h-1V6c0-2.76-2.24-5-5-5S7 3.24 7 6v2H6c-1.1 0-2 .9-2 2v10c0 1.1.9 2 2 2h12c1.1 0 2-.9 2-2V10c0-1.1-.9-2-2-2zM9 6c0-1.66 1.34-3 3-3s3 1.34 3 3v2H9V6zm9 14H6V10h12v10zm-6-3c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2z"
        />
      </svg>
    </div>
    <h1
      className="MuiTypography-root header MuiTypography-h5"
    >
      Sign up
    </h1>
    <form
      className="form"
      noValidate={true}
      onSubmit={[Function]}
    >
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6"
        >
          <div
            className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined Mui-required Mui-required"
              data-shrink={false}
              htmlFor="firstName"
              id="firstName-label"
            >
              First Name
              <span
                aria-hidden={true}
                className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
              >
                 
                *
              </span>
            </label>
            <div
              className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl"
              onClick={[Function]}
            >
              <input
                aria-invalid={false}
                autoComplete="fname"
                autoFocus={true}
                className="MuiInputBase-input MuiOutlinedInput-input"
                disabled={false}
                id="firstName"
                name="firstName"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={true}
                type="text"
              />
              <fieldset
                aria-hidden={true}
                className="PrivateNotchedOutline-root-1 MuiOutlinedInput-notchedOutline"
              >
                <legend
                  className="PrivateNotchedOutline-legendLabelled-3"
                >
                  <span>
                    First Name
                     *
                  </span>
                </legend>
              </fieldset>
            </div>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6"
        >
          <div
            className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined Mui-required Mui-required"
              data-shrink={false}
              htmlFor="lastName"
              id="lastName-label"
            >
              Last Name
              <span
                aria-hidden={true}
                className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
              >
                 
                *
              </span>
            </label>
            <div
              className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl"
              onClick={[Function]}
            >
              <input
                aria-invalid={false}
                autoComplete="lname"
                autoFocus={false}
                className="MuiInputBase-input MuiOutlinedInput-input"
                disabled={false}
                id="lastName"
                name="lastName"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={true}
                type="text"
              />
              <fieldset
                aria-hidden={true}
                className="PrivateNotchedOutline-root-1 MuiOutlinedInput-notchedOutline"
              >
                <legend
                  className="PrivateNotchedOutline-legendLabelled-3"
                >
                  <span>
                    Last Name
                     *
                  </span>
                </legend>
              </fieldset>
            </div>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
        >
          <div
            className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined Mui-required Mui-required"
              data-shrink={false}
              htmlFor="email"
              id="email-label"
            >
              Email Address
              <span
                aria-hidden={true}
                className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
              >
                 
                *
              </span>
            </label>
            <div
              className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl"
              onClick={[Function]}
            >
              <input
                aria-invalid={false}
                autoComplete="email"
                autoFocus={false}
                className="MuiInputBase-input MuiOutlinedInput-input"
                disabled={false}
                id="email"
                name="email"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={true}
                type="text"
              />
              <fieldset
                aria-hidden={true}
                className="PrivateNotchedOutline-root-1 MuiOutlinedInput-notchedOutline"
              >
                <legend
                  className="PrivateNotchedOutline-legendLabelled-3"
                >
                  <span>
                    Email Address
                     *
                  </span>
                </legend>
              </fieldset>
            </div>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
        >
          <div
            className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined Mui-required Mui-required"
              data-shrink={false}
              htmlFor="password"
              id="password-label"
            >
              Password
              <span
                aria-hidden={true}
                className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
              >
                 
                *
              </span>
            </label>
            <div
              className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl"
              onClick={[Function]}
            >
              <input
                aria-invalid={false}
                autoComplete="current-password"
                autoFocus={false}
                className="MuiInputBase-input MuiOutlinedInput-input"
                disabled={false}
                id="password"
                name="password"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={true}
                type="password"
              />
              <fieldset
                aria-hidden={true}
                className="PrivateNotchedOutline-root-1 MuiOutlinedInput-notchedOutline"
              >
                <legend
                  className="PrivateNotchedOutline-legendLabelled-3"
                >
                  <span>
                    Password
                     *
                  </span>
                </legend>
              </fieldset>
            </div>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
        >
          <label
            className="MuiFormControlLabel-root"
          >
            <span
              aria-disabled={false}
              className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-5 MuiCheckbox-root MuiCheckbox-colorPrimary MuiIconButton-colorPrimary"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={null}
            >
              <span
                className="MuiIconButton-label"
              >
                <input
                  className="PrivateSwitchBase-input-8"
                  data-indeterminate={false}
                  onChange={[Function]}
                  type="checkbox"
                  value="allowExtraEmails"
                />
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
              </span>
            </span>
            <span
              className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
            >
              I want to receive inspiration, marketing promotions and updates via email.
            </span>
          </label>
        </div>
      </div>
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-contained submit MuiButton-containedPrimary MuiButton-fullWidth"
        disabled={false}
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="submit"
      >
        <span
          className="MuiButton-label"
        >
          Sign Up
        </span>
      </button>
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-flex-end"
      >
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <a
            className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-body2 MuiTypography-colorPrimary"
            href="#"
            onBlur={[Function]}
            onFocus={[Function]}
          >
            Already have an account? Sign in
          </a>
        </div>
      </div>
    </form>
  </div>
  <div
    className="MuiBox-root MuiBox-root-9"
  >
    <p
      className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
    >
      Copyright © 
      <a
        className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit"
        href="https://material-ui.com/"
        onBlur={[Function]}
        onFocus={[Function]}
      >
        Your Website
      </a>
       
      2021
      .
    </p>
  </div>
</main>
`;

exports[`render correctly date component 1`] = `
<main
  className="MuiContainer-root MuiContainer-maxWidthXs"
>
  <div
    className="paper"
  >
    <div
      className="MuiAvatar-root MuiAvatar-circle avatar MuiAvatar-colorDefault"
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M18 8h-1V6c0-2.76-2.24-5-5-5S7 3.24 7 6v2H6c-1.1 0-2 .9-2 2v10c0 1.1.9 2 2 2h12c1.1 0 2-.9 2-2V10c0-1.1-.9-2-2-2zM9 6c0-1.66 1.34-3 3-3s3 1.34 3 3v2H9V6zm9 14H6V10h12v10zm-6-3c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2z"
        />
      </svg>
    </div>
    <h1
      className="MuiTypography-root header MuiTypography-h5"
    >
      Sign up
    </h1>
    <form
      className="form"
      noValidate={true}
      onSubmit={[Function]}
    >
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6"
        >
          <div
            className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined Mui-required Mui-required"
              data-shrink={false}
              htmlFor="firstName"
              id="firstName-label"
            >
              First Name
              <span
                aria-hidden={true}
                className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
              >
                 
                *
              </span>
            </label>
            <div
              className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl"
              onClick={[Function]}
            >
              <input
                aria-invalid={false}
                autoComplete="fname"
                autoFocus={true}
                className="MuiInputBase-input MuiOutlinedInput-input"
                disabled={false}
                id="firstName"
                name="firstName"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={true}
                type="text"
              />
              <fieldset
                aria-hidden={true}
                className="PrivateNotchedOutline-root-1 MuiOutlinedInput-notchedOutline"
              >
                <legend
                  className="PrivateNotchedOutline-legendLabelled-3"
                >
                  <span>
                    First Name
                     *
                  </span>
                </legend>
              </fieldset>
            </div>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6"
        >
          <div
            className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined Mui-required Mui-required"
              data-shrink={false}
              htmlFor="lastName"
              id="lastName-label"
            >
              Last Name
              <span
                aria-hidden={true}
                className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
              >
                 
                *
              </span>
            </label>
            <div
              className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl"
              onClick={[Function]}
            >
              <input
                aria-invalid={false}
                autoComplete="lname"
                autoFocus={false}
                className="MuiInputBase-input MuiOutlinedInput-input"
                disabled={false}
                id="lastName"
                name="lastName"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={true}
                type="text"
              />
              <fieldset
                aria-hidden={true}
                className="PrivateNotchedOutline-root-1 MuiOutlinedInput-notchedOutline"
              >
                <legend
                  className="PrivateNotchedOutline-legendLabelled-3"
                >
                  <span>
                    Last Name
                     *
                  </span>
                </legend>
              </fieldset>
            </div>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
        >
          <div
            className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined Mui-required Mui-required"
              data-shrink={false}
              htmlFor="email"
              id="email-label"
            >
              Email Address
              <span
                aria-hidden={true}
                className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
              >
                 
                *
              </span>
            </label>
            <div
              className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl"
              onClick={[Function]}
            >
              <input
                aria-invalid={false}
                autoComplete="email"
                autoFocus={false}
                className="MuiInputBase-input MuiOutlinedInput-input"
                disabled={false}
                id="email"
                name="email"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={true}
                type="text"
              />
              <fieldset
                aria-hidden={true}
                className="PrivateNotchedOutline-root-1 MuiOutlinedInput-notchedOutline"
              >
                <legend
                  className="PrivateNotchedOutline-legendLabelled-3"
                >
                  <span>
                    Email Address
                     *
                  </span>
                </legend>
              </fieldset>
            </div>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
        >
          <div
            className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined Mui-required Mui-required"
              data-shrink={false}
              htmlFor="password"
              id="password-label"
            >
              Password
              <span
                aria-hidden={true}
                className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
              >
                 
                *
              </span>
            </label>
            <div
              className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl"
              onClick={[Function]}
            >
              <input
                aria-invalid={false}
                autoComplete="current-password"
                autoFocus={false}
                className="MuiInputBase-input MuiOutlinedInput-input"
                disabled={false}
                id="password"
                name="password"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={true}
                type="password"
              />
              <fieldset
                aria-hidden={true}
                className="PrivateNotchedOutline-root-1 MuiOutlinedInput-notchedOutline"
              >
                <legend
                  className="PrivateNotchedOutline-legendLabelled-3"
                >
                  <span>
                    Password
                     *
                  </span>
                </legend>
              </fieldset>
            </div>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
        >
          <label
            className="MuiFormControlLabel-root"
          >
            <span
              aria-disabled={false}
              className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-5 MuiCheckbox-root MuiCheckbox-colorPrimary MuiIconButton-colorPrimary"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={null}
            >
              <span
                className="MuiIconButton-label"
              >
                <input
                  className="PrivateSwitchBase-input-8"
                  data-indeterminate={false}
                  onChange={[Function]}
                  type="checkbox"
                  value="allowExtraEmails"
                />
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
              </span>
            </span>
            <span
              className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
            >
              I want to receive inspiration, marketing promotions and updates via email.
            </span>
          </label>
        </div>
      </div>
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-contained submit MuiButton-containedPrimary MuiButton-fullWidth"
        disabled={false}
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="submit"
      >
        <span
          className="MuiButton-label"
        >
          Sign Up
        </span>
      </button>
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-flex-end"
      >
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <a
            className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-body2 MuiTypography-colorPrimary"
            href="#"
            onBlur={[Function]}
            onFocus={[Function]}
          >
            Already have an account? Sign in
          </a>
        </div>
      </div>
    </form>
  </div>
  <div
    className="MuiBox-root MuiBox-root-9"
  >
    <p
      className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
    >
      Copyright © 
      <a
        className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit"
        href="https://material-ui.com/"
        onBlur={[Function]}
        onFocus={[Function]}
      >
        Your Website
      </a>
       
      2021
      .
    </p>
  </div>
</main>
`;
